triage-process.md

```diff
@@ -0,0 +1,59 @@
+# High Level Proposal:
+1. Define and publish a process to handle issues in a structured manner.
+    1. Issue Handling Process
+    1. Issue Stages
+    1. Roles
+    1. SLA
+
+## Process Definition
+
+### Issue Handling Process
+1. All incoming issues and PR’s should be as. These are items that a Triager would look at on a daily basis
+1. Every issue triaged should be assessed for its workability. If assessing the workability of the issue requires more information, the Triager would collaborate with the OP until workability is determined.
+    1. Workability Classification:
+        1. New Feature Request
+        1. Enhancement to Existing Feature
+        1. Bug
+1. Workable Issues should be prioritized and Labeled as one of 3 priorities
+    * P0
+    * P1
+    * P2
+1. Prioritized Issues (and unworked) issues should be labeled as ‘Ready for Work’. This should signal to any contributor willing to help that they can work on this issue.
+1. Once a contributor is working on this issue, the issue should be:
+    1. Assigned to him
+    1. Labeled as ‘In Progress’
```

Comment:
no 'In Progress' label is not required. If it is assigned then it is in progress.

Reply:
Its not a label. I think you might be reading the column incorrectly.  See action lablel column far right in the same row.

Reply:
table is clear but the above line contradicts the table. If I read 'the issue should be Labeled as "In Progress"' I cannot infer anything but to apply a label called "In Progress". 
I would rephrase the above to
```
1. Once a contributor is working on this issue, the issue should be:
    1. Assigned to them. (Assigned issues are considered to be in "In Progress" stage.)
```

Reply:
'ready-for-work' and 'in-progress' should not be called labels - they're stages.

Reply:
done